README.md

```diff
@@ -15,7 +15,8 @@
 * [QA Dashboard](https://dashboard.heroku.com/apps/s22-6pm-3-happycows-qa)
 
 6pm-4:
-* [QA](https://team04-w22-6pm-4-qa.herokuapp.com/)
+* [QA](https://s22-6pm-4-happycows-qa.herokuapp.com/)
+* [QA-Dashboard](https://dashboard.heroku.com/apps/s22-6pm-4-happycows-qa)
```

Comment:
Part of this task was to give staff access to this dashboard, but that hasn't happened yet:

<img width="691" alt="image" src="https://user-images.githubusercontent.com/1119017/170844320-0a5d0609-99b7-491e-a131-2ed93e8453e7.png">

Reply:
Fixed now - staff emails have been added as collaborators to the qa deployment.